ecosystem/sep-TBD.md

```diff
@@ -0,0 +1,155 @@
+
+## Preamble
+
+```
+SEP: To Be Assigned
```

Comment:
@Shaptic Can you assign this 0039, and rename the file to match, and then add the file to the list of SEPs in the ecosystem folder's README.md?

Reply:
Changes in https://github.com/stellar/stellar-protocol/pull/1140/commits/ea1dfc6de64ad04b655d7320774006cbedd1c0cb and subsequent rename in https://github.com/stellar/stellar-protocol/pull/1140/commits/1d367aefcf6b47021f9a8b17e6c369275d1c1ebc!